docs/design/notes-todo-split.md

```
@@ -0,0 +1,390 @@
+    created_at REAL NOT NULL
+);
+
+-- Member + revision tables shared via shared_collaboration module
+```
+
+### 1c. Shared Collaboration Module (extracted, not duplicated)
+
+Both stores use identical member/permission/revision patterns. Extract these
+into a reusable module at `tinyagentos/collaboration/`:
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)